VMBackup/main/handle.py

```diff
@@ -277,6 +277,8 @@ def daemon():
             backup_logger.log(errMsg, True, 'Error')
 
         freezer = FsFreezer(patching= MyPatching, logger = backup_logger, hutil = hutil)
+        backup_logger.log(freezer.file_exists, True, 'Info')
```

Comment:
Please add some text as to what variable is being printed. For example {FreezeFileExists: true}. Currently only variable value is printed.